### PR TITLE
Update example to work in Rails console

### PR DESCRIPTION
**Pull request by RogerPodacter**
# Fixes # 

### Checklist
- [ x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [ x] I have read the [Contribution Guide] and my PR follows them.
- [ x] I updated my branch with the master branch.
- [ x] I have added tests that prove my fix is effective or that my feature works
- [ x] I have added necessary documentation about the functionality in the appropriate .md file
- [ x] I have added in line documentation to the code I modified

### Short description of what this PR does:
Because `Mail` is used by Rails the "hello world" example doesn't work in the Rails console out of the box. This change fixes that and is also consistent with the existing reference to `SendGrid::API` which is also strictly speaking unnecessary if one is including this module in a class.

Perhaps there is another way to handle this but my view FWIW is that it's supremely important that simplest example work in the Rails console with no changes.

### CODE REVIEW

**Comment by SendGridDX** [comment]
[CLA assistant check](https://cla.sendgrid.com/sendgrid/sendgrid-ruby?pullRequest=306) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.sendgrid.com/sendgrid/sendgrid-ruby?pullRequest=306) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.sendgrid.com/check/sendgrid/sendgrid-ruby?pullRequest=306) it.</sub>

File: README.md [hunk 1]
```diff
@@ -83,11 +83,11 @@ The following is the minimum needed code to send an email with the [/mail/send H
 require 'sendgrid-ruby'
 include SendGrid
 
-from = Email.new(email: 'test@example.com')
-to = Email.new(email: 'test@example.com')
+from = SendGrid::Email.new(email: 'test@example.com')
+to = SendGrid::Email.new(email: 'test@example.com')
 subject = 'Sending with SendGrid is Fun'
 content = Content.new(type: 'text/plain', value: 'and easy to do anywhere, even with Ruby')
```
Comment (thinkingserious) [inline README.md]: Why not `SendGrid::Content` here?

Comment (RogerPodacter) [reply README.md]: Good point! Fixed.

**Comment by thinkingserious** [comment]
Hello @RogerPodacter,<br /><br/>Thanks again for the PR!<br />                                              <br />We want to show our appreciation by sending                                              you some swag. Could you please fill out                                              <a href="https://dx.sendgrid.com/swag">this form</a> so we can send it to you? Thanks!                                             <br /><br />Team SendGrid DX

**Comment by RogerPodacter** [comment]
Done! Thank you and my pleasure!

**Pull request merged**